linchpin/api/repository_controls/test.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This file should probably go in the tests directory, and have a unit test calling it.